tools/installer/run-installer.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Add documentation here:
Like it will be installing XED, tis camera, ROS. 
Anything, in particular, we need to be careful about and stuff.

Reply:
Or you can add all this in Readme